dev/fattest.simplicity/src/componenttest/topology/utils/tck/TCKResultsInfo.java

```diff
@@ -263,13 +271,12 @@ public String getReadableRepeatName() {
         String readableRepeatName = getRepeat();
 
         if (readableRepeatName.contains("FeatureReplacementAction")) {
-            readableRepeatName = readableRepeatName.replaceAll("FeatureReplacementAction.*REMOVE", "remove")
+            readableRepeatName = readableRepeatName.replaceAll("FeatureReplacementAction.*REMOVE ", "remove_")
                             .replaceAll("\\[", "")
                             .replaceAll("\\]", "")
-                            .replaceAll("ADD", "add")
-                            .replaceAll("  ", " ")
+                            .replaceAll(" ADD ", "_add_")
                             .replaceAll(",", "-")
-                            .replaceAll(" ", "_");
+                            .replaceAll(" ", "");
         }
```

Comment:
The need to change this highlights an issue - this was added in the previous PR, but it's not clear from the method name what it's really for. I'm unsure it does make the repeat name any more readable - but it looks like it's used as part of the directory and filename for the results?

We should probably rename the method and add some Javadoc to explain what it's supposed to do. As it stands, someone could come along and change it to make the output "more readable" and likely break something.